web/.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL=https://cors-anywhere.herokuapp.com/https://api-dot-gpay-group-buy.an.r.appspot.com/
```

Comment:
Shouldn't backend url for development be localhost:5000? Otherwise, you cannot see the changes that are not yet deployed.

Reply:
Sighh I wanted to make it such that FE and BE dev are independent. Guess it's super hard to do since the endpoint that we want would be stuck in some other PR and not in prod. 🤣

Reply:
Ohh okay. Also, for cors, I think we can add cors library to express so we don't have to use the cors-anywhere?
https://expressjs.com/en/resources/middleware/cors.html

Reply:
Haha I was thinking it's just for local dev anyway so I just did a quick fix. But anw it depends if we are keeping it as prod BE or localhost BE?

Reply:
ehh ya it was more of an FYI